attributes/default.rb

```diff
@@ -80,8 +80,8 @@
 default['apache']['default_site_name'] = 'default'
 
 # Where the various parts of apache are
-case node['platform']
-when 'redhat', 'centos', 'scientific', 'fedora', 'amazon', 'oracle'
```

Comment:
these are explicitly `platform` and not `platform_family` as we do not have `platform_family` in serverspec, so we can't match platform fixtures and this fails. It is much easier to keep this this way until we add platform_family support to the integration test fixtures.